src/guides/wallet.md

```diff
@@ -0,0 +1,164 @@
+## Seting up a Handshake Wallet 
+
+
+`hsd` offers a powerful, modular way to create and manage handshake wallets. In this guide, I'll walk you through the concepts and features you'll need to know about to get started.
+
+### The Basics
+If you're a seasoned bitcoiner, you can probably skim this section or skip straight ahead to the <a href='#examples'> Examples </a>section. But if you're relatively new or just want a refresher, this section will help you understand how wallets actually work.
+
+### Wallets
+In the most basic sense, a handshake wallet is data that enables you to receive and spend HNS, place bids on names and update resource records for your names. `hsd` implements the latest specifications for structuring wallets that are easy to backup, easy to restore, and that work just as well for a novice making their first transactions as for a business with millions of users.
+
+### Keys to the Game
+If you want to transact with HNS, you'll need keys. Each handshake address is associated with a particular key, and wallets are made up of many different keys. Keys consist of both a private key and a public key. The private key is required for spending and is extremely sensitive information, while a public key can be used to receive HNS  and monitor a particular address. If you want to learn more about how this works, read up on Public-Key Cryptography.
```

Comment:
nit: The line, "while a public key can be used to receive HNS  and monitor" has a double space here: "HNS  and"